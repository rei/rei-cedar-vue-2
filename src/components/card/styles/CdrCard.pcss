/* ==========================================================================
  # CdrCard
  
  All values should map to variables in
  vars/CdrCard.vars.pcss
  in order to allow for theming

  TOC:

    :Base Styles
========================================================================== */

.cdr-card {
  background-color: $card-background-color;
  border-width: $card-border-width;
  border-style: $card-border-style;
  border-color: $card-border-color;
  border-radius: $card-border-radius;
  box-shadow: $card-box-shadow;
  color: $card-color;
  width: $card-width;
}
