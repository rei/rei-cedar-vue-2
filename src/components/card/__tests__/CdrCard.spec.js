import { mount } from '@vue/test-utils';
import CdrCard from 'componentdir/card/CdrCard';

describe('CdrCard', () => {
  test('renders correctly', () => {
    const wrapper = mount(CdrCard);
    expect(wrapper.element).toMatchSnapshot();
  });
});
