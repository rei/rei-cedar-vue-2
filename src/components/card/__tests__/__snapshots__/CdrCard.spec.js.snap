// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CdrCard renders correctly 1`] = `
<article
  class="cdr-card"
/>
`;
